Get-DevConsole/Install-DevConsole.ps1

```diff
@@ -0,0 +1,35 @@
+# Progress indicator
+$ProgressPreference = "SilentlyContinue"
+
+# Set strict mode.
+Set-StrictMode -Version 2.0
+ 
+# Set default ErrorAction to Stop.
+$ErrorActionPreference = "Stop"
+
+#Import GetDevConsole Module
+Try
+{
+    $scriptPath = $script:MyInvocation.MyCommand.Path
+    $scriptDirectory = Split-Path $scriptPath
+
+    Import-Module "$scriptDirectory\GetDevConsole.psm1" -Force -DisableNameChecking
+    Import-Module "$scriptDirectory\InstallDevConsole.psm1" -Force -DisableNameChecking
+    Import-Module "$scriptDirectory\..\BTModules\BTSupport.psm1" -Force -DisableNameChecking
+}
+Catch
+{
+    Write-Host "Failed to import required modules: Breaking script" -ForegroundColor Red
+    exit

```

Comment:
The use of "exit" being a bad practice is mostly a C++ concept, but is still relevant to Powershell. Using "exit" will close the entire window, instead of just leaving the script you are executing. I've always been taught that "exit" is kind of like kicking your ex out of the house the moment you break up and never letting her back in to get her stuff. There are much more humane ways of going about these things, like using "return $false" or even just "return". Give the poor lady a chance to gather her belongings before kicking her to the curb.